pokedex_async_redux/lib/state/action/action.dart

```diff
@@ -0,0 +1,14 @@
+import 'package:async_redux/async_redux.dart';
+import 'package:pokedex_async_redux/state/app_state.dart';
+
+abstract class LoadingAction extends ReduxAction<AppState> {
```

Comment:
you forgot to add a concise documentation 😝

remember, this is mandtoy right? 😉